ee/utils_ee.go

```diff
@@ -31,6 +31,7 @@ func GetEEFeaturesList() []string {
 	var ee []string
 	if len(worker.Config.HmacSecret) > 0 {
 		ee = append(ee, "acl")
+		ee = append(ee, "multi_tenancy")
```

Comment:
Please make sure we don't port this to 20.11. This should be in 21.03+ versions.

Reply:
Yeah. I will not cherry-pick it.